doc/api.md

```diff
@@ -140,6 +140,14 @@ The `event` parameter is a String object with the name of the event.
 The `listener` parameter is a Function object with one argument that will be notified when the event occurs with data related to the event.
 
 The following events are currently supported:
+* **avatarChanged** - event notifications about avatar
+changes. The listener will receive an object with the following structure:
+```javascript
+{
+"id": id,                 // the id of the participant that changed his avatar.
```

Comment:
I don't like the misalignment of the comments here. I generally don't try to enforce such styling but I think this one is worth resolving because it's in a readme whose audience is broader than the rest of the codebase. Maybe get rid of the extra spacing here.

Reply:
Fixed!